src/containers/namespace-detail/namespace-detail.tsx

```diff
@@ -377,7 +377,23 @@ export class NamespaceDetail extends React.Component<IProps, IState> {
           !collection.deprecated,
           this.context.selectedRepo,
         )
-          .then(() => this.loadCollections())
+          .then(() => {
+            let title;
```

Comment:
also, what's up with the `let title; expression ? (title = foo) : (title = bar)` construct?

```suggestion
            const title = !collection.deprecated ? t`Collection ${collection.name} has been deprecated.` : t`Collection "${collection.name}" has been successfully undeprecated.`;
```